py/strato/common/log/log2text.py

```diff
@@ -56,25 +65,62 @@ def __init__(self, relativeTime, withThreads, showFullPaths, noDebug, microsecon
             (NORMAL_COLOR if useColors else '') + \
             ("(%(pathname)s:%(lineno)s)" if showFullPaths else "(%(module)s::%(funcName)s:%(lineno)s)")
 
-    def process(self, obj):
-        if obj['levelno'] < self._minimumLevel:
+    def process(self, line, logPath, logConf):
+        if os.path.basename(logPath).endswith(".stratolog"):
```

Comment:
not good enough. can we at least try to check whether its *.stratolog*

Same about exceptions.log*